### PR TITLE
two new examples

### PR DESCRIPTION
**Pull request by stla**
Added two examples which use a JavaScript function in the options.

### CODE REVIEW

**Review by yihui — APPROVED** [review]
Thanks!

File: inst/examples/DT-searchExact/server.R [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      "  $inputs.off('keyup search input').on('keyup', function() {",
+      "    var value = $(this).val();",
+      "    if(value !== '') {",
+      "      value = '^' + value + '$';",
+      "      var index = 1 + $inputs.index(this);", # add one for rownames column
+      "      var column = table.column(index);",
+      "      column.search(value, true, false).draw();",
+      "    }",
+      "  });",
+      "}"
```
Comment (yihui) [inline inst/examples/DT-searchExact/server.R:41]: For large amounts of JS code, I wonder if it'd better to write them in a separate JS file and read the lines here. That might make the JS code easier to read and develop.

Comment (stla) [reply inst/examples/DT-searchExact/server.R:41]: But is it possible to make this JS file visible with the `showcase` option of Shiny?

Comment (yihui) [reply inst/examples/DT-searchExact/server.R:41]: Good point. No.

Okay, I'll just merge it as is. Thanks!

**Pull request merged**